package.json

```diff
@@ -12,21 +12,21 @@
     "xlsx",
     "xls",
     "doc",
-    "docx"
+    "docx",
+    "pdf"
   ],
   "contributors": [
     "René Rössler <rene@freshx.de>"
```

Comment:
@squalsoft Please add yourself to the list of contributors! I will accept the merge request afterwards because it looks very good to me. Thanks for the test case rework!